cmd/openshift-install/version.go

```diff
@@ -33,6 +34,14 @@ func runVersionCmd(cmd *cobra.Command, args []string) error {
 	if image, err := releaseimage.Default(); err == nil {
 		fmt.Printf("release image %s\n", image)
 	}
-	fmt.Printf("release architecture %s\n", version.DefaultArch())
+	releaseArch, err := version.ReleaseArchitecture()
+	if err != nil {
+		return err
+	}
+	fmt.Printf("release architecture %s\n", releaseArch)
+	if strings.Contains(releaseArch, "multi") || strings.Contains(releaseArch, "unknown") {
+		fmt.Printf("default architecture %s\n", version.DefaultArch())
```

Comment:
@r4f4 Thought it might be a good idea to print default arch when release arch is unknown? This would give some indication of what the installer will do?

Reply:
+1 from me.